Tavisca.Bootcamp.LanguageBasics.Exercise3/Program.cs

```diff
@@ -1,10 +1,25 @@
 ﻿using System;
 using System.Linq;
-
+using System.Collections.Generic;
 namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
 {
+    
     public static class Program
     {
+        public class Food{
+            public int pro;

```

Comment:
You should use Properties here, rather than using public variables.
Also make sure that you make 'setter' private for the properties which you are gonna set in constructor.
Also naming variable 'protein' will make it more readable.